lib/public/Http/Client/IClientService.php

```diff
@@ -33,8 +33,9 @@
  */
 interface IClientService {
 	/**
+	 * @param bool|null $useCookieJar - keep cookies between requests.
 	 * @return IClient
-	 * @since 8.1.0
+	 * @since 8.1.0 - Since Nextcloud 27.0.0 cookies can be kept with $useCookieJar.
 	 */
-	public function newClient(): IClient;
+	public function newClient(?bool $useCookieJar = false): IClient;
```

Comment:
```suggestion
	public function newClient(bool $useCookieJar = false): IClient;
```

No need for null, boolean with on/off is good enough

Reply:
But doesn't that break the current usage of the API?
If the `$useCookieJar` parameter is required we will need to provide it everywhere `newClient` is called, won't we?

Reply:
It is not required, the `= false` means it falls back to boolean false